docs/index.rst

```diff
@@ -36,22 +36,27 @@ In this documentation
 
    .. grid-item:: :doc:`Tutorial <tutorial/index>`
 
-      **Start here**: a hands-on introduction to Real-time Ubuntu for new users
+      **Start here**: a hands-on introduction to Real-time Ubuntu for new users.
 
    .. grid-item:: :doc:`How-to <how-to/index>`
 
-      **Step-by-step guides** covering key operations and common tasks
+      **Step-by-step guides** covering key operations and common tasks,
+      including :doc:`installation <how-to/enable-real-time-ubuntu>` and
+      :doc:`configuration <how-to/cpu-boot-configs>`.
```

Comment:
I removed the links for now but kept the description.
It would be more useful if there were a bigger number of topics for each category and they had their own landing pages.

Thanks.